app/src/main/java/com/example/sharingang/SetPriceActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.sharingang
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+
+class SetPriceActivity : AppCompatActivity() {
+    companion object {
+        const val EXTRA_PRICE = "com.example.sharingang.EXTRA_PRICE"
+        var CHOSEN_CURRENCY = "USD"
```

Comment:
This isn't ever modified so it really shouldn't be a var. I think `const val` would be the most appropriate here.